### PR TITLE
create layout l007  as  sections `<ToDo />`

### PR DESCRIPTION
**Pull request by Andresbl123**
The ToDo section is implemented, where simulated data is consumed from a database to display the corresponding information. This section is aligned with the request page, and the information is obtained through a parameter passed by the URL.

In this PR, the layout adjustment of each of the sections of the request page is made. This is done by adding a new prop to the `<Fieldset />` component, so several files are superficially modified.

image

### CODE REVIEW

**Review by novadeveloper14 — APPROVED** [review]

File: src/pages/board/outlets/boardlayout/ToDo/index.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+  return (
+    <Fieldset
+      title="Por hacer"
+      descriptionTitle={
+        data.nnasocia ? capitalizeFirstLetterEachWord(data?.nnasocia) : ""
+      }
+      heigthFieldset="284px"
+    >
+      <Stack direction="column" gap="6px">
```
Comment (carin91) [inline src/pages/board/outlets/boardlayout/ToDo/index.tsx:56]: Adjust the gap with inube tokens

File: src/pages/board/outlets/boardlayout/ToDo/index.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+              loading={loading || false}
+              type="submit"
+            >
+              {label || "Enviar"}
+            </Button>
+          </Stack>
+        </Stack>
+        <Divider />
+        <Stack gap={inube.spacing.s200} alignItems="center" padding="s100 s0">
+          <Select
```
Comment (carin91) [inline src/pages/board/outlets/boardlayout/ToDo/index.tsx:87]: Accordingly to the wireframes this is not a select, does it is a textfield?
image

File: src/pages/board/outlets/boardlayout/ToDo/index.tsx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+            id="gestorComercial"
+            name="gestorComercial"
+            label="Gestor Comercial"
+            placeholder="Gestor Comercial"
+            options={optionsSelectGestorComercial}
+          />
+
+          <Select
```
Comment (carin91) [inline src/pages/board/outlets/boardlayout/ToDo/index.tsx:95]: Accordingly to the wireframes this is not a select, does it is a textfield?
image

File: src/routes/board.tsx [hunk 4]
```diff
@@ -8,7 +8,9 @@ import { Requirements } from "@pages/board/outlets/boardlayout/Requirements";
 import { dataRequirements } from "@pages/board/outlets/boardlayout/Requirements/config";
 import { PromissoryNotes } from "@pages/board/outlets/boardlayout/PromissoryNotes";
 import { Postingvouchers } from "@pages/board/outlets/boardlayout/Postingvouchers";
-import { Approvals } from "@pages/board/outlets/financialReporting/approvals";
+import { Approvals } from "@src/pages/board/outlets/financialReporting/Approvals";
```
Comment (carin91) [inline src/routes/board.tsx:11]: why do you change this import?, the best option would be starting with @pages, does it not?

File: src/pages/board/outlets/boardlayout/ToDo/index.tsx [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+      }
+      heigthFieldset="284px"
+    >
+      <Stack direction="column" gap="6px">
+        <Stack>
+          <Text>{data?.n_Descr_Tarea}</Text>
+        </Stack>
+        <Stack gap={inube.spacing.s200} padding="s100 s0" alignItems="center">
+          <Stack width="340px">
+            <Select
```
Comment (carin91) [inline src/pages/board/outlets/boardlayout/ToDo/index.tsx:62]: I'm not really like the fieldset behavior related to this select.
Could you validate some other options with Leonardo about it?


https://github.com/selsa-inube/crediboard/assets/123512032/80ad801e-6414-4886-8500-7c3581b6f650

**Review by carin91 — APPROVED** [review]

File: src/pages/board/outlets/boardlayout/ToDo/index.tsx (hunk 1 above)
Comment (Andresbl123) [reply src/pages/board/outlets/boardlayout/ToDo/index.tsx:56]: resuelto

File: src/routes/board.tsx (hunk 4 above)
Comment (Andresbl123) [reply src/routes/board.tsx:11]: resuelto

File: src/pages/board/outlets/boardlayout/ToDo/index.tsx (hunk 3 above)
Comment (Andresbl123) [reply src/pages/board/outlets/boardlayout/ToDo/index.tsx:95]: resuelto

**Pull request merged**